config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This might change based on some of the upcoming stories from @Birdrock, but this seems sufficient for now. 👍